Formula/zim.rb

```diff
@@ -3,7 +3,8 @@ class Zim < Formula
   homepage "https://zim-wiki.org/"
   url "https://github.com/jaap-karssenberg/zim-desktop-wiki/archive/0.73.2.tar.gz"
   sha256 "f002e63d7c131e317225c2912d5cc2065da90448f7ebd330ad4931a57fa45eda"
-  license "GPL-2.0"
+  license "GPL-2.0-only"
```

Comment:
```suggestion
  license "GPL-2.0-or-later"
```

The [license header](https://github.com/zim-desktop-wiki/zim-desktop-wiki/blob/b6f92c40477992a370c90a16ef920148c3ca372f/zim/__init__.py#L2-L12) I'm seeing:

```python
# Copyright 2008-2018 Jaap Karssenberg <jaap.karssenberg@gmail.com>

# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version.

# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
```

The `or (at your option) any later version.` part indicates `-or-later`

Reply:
👍 I just looked at the license file, which didn't indicate the or-later.

Reply:
Yeah, the GNU licenses are tricky. They all follow the same format though. A general rule is to look at the COPYING file, the README, and a few source files (e.g. main.c). If they have a license header, it will either say `(at your option) any later version.` (in which case you use `-or-later`) or it will just say `version x of the License` (in which case you use `-only`).

Sometimes, there's no indication either way. When this happens we just go with the `-only`.